Makefile

```diff
@@ -246,7 +246,15 @@ bundle-push: ## Push the bundle image.
 
 .PHONY: deploy-bundle
 deploy-bundle: operator-sdk bundle-build bundle-push
-	$(OPERATOR_SDK) run bundle $(BUNDLE_IMG)
+	$(OPERATOR_SDK) run bundle $(BUNDLE_IMG)  -n $(OPERATOR_NAMESPACE)
+
+.PHONY:	create-namespace
+create-namespace: ## Create namespace
+	kubectl create namespace $(OPERATOR_NAMESPACE)
+
+.PHONY: upgrade-bundle
+upgrade-bundle: bundle-build bundle-push ## Upgrade bundle
```

Comment:
Isn't it inconsistent with other bundle* targets?

Reply:
Should it have dependency on operator-sdk like deploy?

Reply:
My assumption was we have `operator-sdk` in deploy-bundle and it will available for next cmd upgrade-bundle as well. If it's required to have it as a dependency then I'll add it.

Reply:
> Isn't it inconsistent with other bundle* targets?

It consistent with `deploy-bundle` . if we want it `bundle-upgrade` then we should also change `deploy-bundle` as `bundle-deploy`.

Reply:
> > Isn't it inconsistent with other bundle* targets?
> 
> It consistent with `deploy-bundle` . if we want it `bundle-upgrade` then we should also change `deploy-bundle` as `bundle-deploy`.

yes, agree. Well, I would prefer all of the target consistent, but no problem if it is ok for others.

Reply:
> My assumption was we have `operator-sdk` in deploy-bundle and it will available for next cmd upgrade-bundle as well. If it's required to have it as a dependency then I'll add it.

well, it's some implicit assumption which is no problem to avoid so I would vote for adding the dependency.